### ISSUE TITLE
How do I use CTC on the DIGITS

### ISSUE THREAD

**Issue by littlepai** [NVIDIA/DIGITS#2219]
I found that the DIGITS provided some standard models, but there was nothing about CTC loss, When I tried to build my own, I found that I was familiar with PyTorch, but not the torch(by Lua). I was familiar with TensorFlow, But the built-in TensorFlow in DIGITS is different.

So.
1. I want to use CTC loss in DIGITS by PyTorch, how?
2. How do I use regular python TensorFlow in DIGITS?
3. Is the TensorFlow pre-training model universal?

thank you

**Comment by littlepai** [comment]
> I found that the DIGITS provided some standard models, but there was nothing about CTC loss, When I tried to build my own, I found that I was familiar with PyTorch, but not the torch(by Lua). I was familiar with TensorFlow, But the built-in TensorFlow in DIGITS is different.
> 
> So.
> 
> 1. I want to use CTC loss in DIGITS by PyTorch, how?
> 2. How do I use regular python TensorFlow in DIGITS?
> 3. Is the TensorFlow pre-training model universal?
> 
> thank you
